lib/components/Select.js

```diff
@@ -46,7 +47,14 @@ const Select = ({
   let Parent = SelectInput;
 
   if (isCreateable) {
+    Parent = Creatable;
+  }
+
+  if (otherProps.loadOptions) {
     Parent = AsyncCreatable;
+  } else if (otherProps.defaultOptions) {
+    otherProps.options = otherProps.defaultOptions;
+    otherProps.defaultOptions = null;
   }
```

Comment:
@mohit-s96 _a

As discussed, I think we should update the docs for `Creatable` to use only `options` prop instead. The `defaultOptions` could be a boolean or a list of options, so this could cause some bugs.

@amaldinesh7 LMK your thoughts

Reply:
Yeah, we can update the stories to use `options` instead of `defaultOptions` and remove any code related to `defaultOptions` from select component